backend/src/test/java/com/crm/service/impl/CustomerServiceImplTest.java

```diff
@@ -161,23 +135,81 @@ public void shouldThrowConflictWhenCustomerIsDuplicated() {
     }
 
     @Test
-    public void shouldThrowConflictWhenPhoneExists() {
-        customerRequest = CustomerRequest.builder()
+    public void shouldNotAddCustomerAndThrowConflictWhenPhoneExists() {
+        var customerRequest = CustomerRequest.builder()
```

Comment:
We need to discuss about `var `declaration in project. But for me `var `in test is enough